assets/stylesheets/base/_mixins.scss

```diff
@@ -0,0 +1,9 @@
+@mixin border-bottom($width: 1px, $style: solid, $color: $underline-color) {
```

Comment:
I totally understand. The only reason I included this was that it was a carry-over from embed - there's a lot of bottom border action going on and I was anticipating bringing in all of those styles.